### PR TITLE
display-visit-time-12-hour-clock

### PR DESCRIPTION
**Pull request by jonhindle**
### Context

> Does this issue have a Trello card?

https://trello.com/c/dW459ORh

> If this is an issue, do we have steps to reproduce?

Navigate to profile page and scroll to visits section

### Intent

> What changes are introduced by this PR that correspond to the above card?

Time displaying for visits
Centralised lowercase am/pm times and removed the extra CSS and nunjucks lowercase transformations

> Would this PR benefit from screenshots?

### Considerations

> Is there any additional information that would help when reviewing this PR?

You'll need a user with an upcoming visit

> Are there any steps required when merging/deploying this PR?

N/A

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/auth/middleware.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ function isPrisonerId(id) {
   return pattern.test(id);
 }
 
+// G1727GV, G2320VD, G2732GG, G2168GG, G9374GG
```
Comment (jonhindle) [inline server/auth/middleware.js:40]: test accounts included for convenience

File: server/views/pages/profile.html [hunk 2]
```diff
@@ -50,7 +50,7 @@ <h1>{{ title }}</h1>
     {% for event in events %}
       <div class="govuk-!-padding-bottom-4">
         <strong>{{event.description}}</strong><br/>
-        {{event.startTime | lower }} to {{event.endTime | lower }}<br/>
+        {{event.startTime }} to {{event.endTime }}<br/>
```
Comment (jonhindle) [inline server/views/pages/profile.html:53]: now handled in the time format.

File: assets/sass/components/_home-navigation.scss [hunk 3]
```diff
@@ -127,10 +127,6 @@
       text-transform: capitalize;
     }
 
-    & > div > .timetable-time {
-      text-transform: lowercase;
-    }
```
Comment (jonhindle) [inline assets/sass/components/_home-navigation.scss:132]: now handled in the time format.

File: assets/sass/components/_timetable.scss [hunk 4]
```diff
@@ -38,7 +38,6 @@
 }
 
 .timetable-time {
-  text-transform: lowercase;
```
Comment (jonhindle) [inline assets/sass/components/_timetable.scss:41]: now handled in the time format.

File: server/app.js [hunk 5]
```diff
@@ -258,9 +258,7 @@ const createApp = ({
   app.use(
     '/profile',
     createProfileRouter({
-      hubContentService,
       offenderService,
-      prisonerInformationService,
     }),
```
Comment (davidthomas22) [inline server/app.js:262]: are these no longer used by the profile route?

Comment (jonhindle) [reply server/app.js:262]: The `createProfileRouter` definition, only pulls out the offenderService, from what I can see.
https://github.com/ministryofjustice/prisoner-content-hub-frontend/blob/3516af024d1f53b503819d32ba79daf702c02cf9/server/routes/profile.js#L3

**Review by davidthomas22 — APPROVED** [review]
LGTM

File: assets/sass/components/_home-navigation.scss (hunk 3 above)
Comment (andrewrlee) [reply assets/sass/components/_home-navigation.scss:132]: nice

File: server/services/offender/responses/__tests__/nextVisit.spec.js [hunk 6]
```diff
@@ -38,11 +41,13 @@ describe('NextVisit', () => {
     expect(formatted).toStrictEqual(
       {
         hasNextVisit: true,
-        nextVisit: 'Saturday 7 December 2019',
+        nextVisit: 'Saturday 7 December',
```
Comment (andrewrlee) [inline server/services/offender/responses/__tests__/nextVisit.spec.js:44]: assume losing the year here is intentional

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**